### PR TITLE
fix(ThresholdDescription): RHICOMPL-2836 description of threshold

### PR DESCRIPTION
**Pull request by xmicha82**
## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [x] Input Validation
- [x] Output Encoding
- [x] Authentication and Password Management
- [x] Session Management
- [x] Access Control
- [x] Cryptographic Practices
- [x] Error Handling and Logging
- [x] Data Protection
- [x] Communication Security
- [x] System Configuration
- [x] Database Security
- [x] File Management
- [x] Memory Management
- [x] General Coding Practices

### CODE REVIEW

File: src/PresentationalComponents/PolicyDetailsDescription/PolicyDetailsDescription.js [hunk 1]
```diff
@@ -30,7 +30,8 @@ const PolicyDetailsDescription = ({ policy }) => (
           <PolicyThresholdTooltip />
         </Text>
         <Text className="threshold-tooltip" component={TextVariants.p}>
-          {fixedPercentage(policy.complianceThreshold, 1)}
+          {`${fixedPercentage(policy.complianceThreshold, 1)} of rules must be
+        passed for a system to be labeled "Compliant"`}
```
Comment (skateman) [inline src/PresentationalComponents/PolicyDetailsDescription/PolicyDetailsDescription.js:34]: Maybe? Or do you have better inputs from PM?
```suggestion
        passed for a system to be labeled as "Compliant"`}
```

**Comment by skateman** [comment]
I'm a bit concerned that we have no snapshot for this view :thinking:

**Review by bastilian — APPROVED** [review]
LGTM! Thank you @xmicha82!

File: src/PresentationalComponents/PolicyDetailsDescription/PolicyDetailsDescription.js (hunk 1 above)
Comment (bastilian) [reply src/PresentationalComponents/PolicyDetailsDescription/PolicyDetailsDescription.js:34]: This is the text from the mockups.

**Comment by bastilian** [comment]
@skateman The component doesn't really have "behaviour", that is why there is no test. We could add a simple test, but it wouldn't really test anything and just be a snapshot. It would bump coverage slightly.

File: src/PresentationalComponents/PolicyDetailsDescription/PolicyDetailsDescription.js (hunk 1 above)
Comment (xmicha82) [reply src/PresentationalComponents/PolicyDetailsDescription/PolicyDetailsDescription.js:34]: i used the exact wording from [this mock](https://marvelapp.com/prototype/93199gg/screen/86593801)

**Pull request merged**